modules/bench/src/format-utils.js

```diff
@@ -0,0 +1,31 @@
+export function formatSI(number, precision = 3) {
```

Comment:
This diffs as a new file. Is this file being copied or was the intention to move it?

I assume new code is not being added in this restructuring PR, better keep that for a separate PR>

Reply:
This is moved from `formatters.js`. `Bench` is the only class depending on this so it doesn't make sense to export it from core.